scripts/resinhup/run-resinhup-ssh.sh

```diff
@@ -46,11 +46,11 @@ Options:
         Run run-resinhup.sh with --remote . See run-resinhup.sh help for more details.
 
   --hostos-version <HOSTOS_VERSION>
-        Run run-resinhup.sh with --version .  See run-resinhup.sh help for more details.
+        Run run-resinhup.sh with --version . See run-resinhup.sh help for more details.
```

Comment:
It's all over the file. I assumed it's deliberate, to clearly make it stand out. I agree though. @agherzan?